pages/common/az-bicep.md

```diff
@@ -0,0 +1,33 @@
+# az bicep
+
+> Bicep CLI command group.
+> Part of `azure-cli`.
+> More information: <https://docs.microsoft.com/cli/azure/config>.
+
+- Install Bicep CLI:
```

Comment:
Is there a Bicep non-CLI program/app?

Reply:
No... there is only an optional VS Code Extension along with the CLI, but no other programs or GUIs.

Reply:
Also thanks for the suggestions!

Reply:
@navarroaxel Should I remove the "CLI" word from there? In my personal opinion is relevant and it's always referenced as "Bicep CLI".